packages/gatsby/lib/query-runner/route-writer.js

```diff
@@ -13,6 +13,87 @@ import {
 // Loop through all paths and write them out to child-routes.js
 const writeChildRoutes = () => {
   const { program, config, pages } = store.getState()
+
+  // console.log(pages)
+  // Write out routes.json
+  const routesData = _.merge(
```

Comment:
I like it! I'll make that change. Object/array spreading makes reducing quite nice!